server/applicationPattern/applicationPattern/onfModel/models/layerProtocols/HttpClientInterface.js

```diff
@@ -13,9 +13,10 @@ const controlConstruct = require('../ControlConstruct');
 const logicalTerminationPoint = require('../LogicalTerminationPoint');
 const layerProtocol = require('../LayerProtocol');
 const onfPaths = require('../../constants/OnfPaths');
+const ForwardingConstruct = require('onf-core-model-ap/applicationPattern/onfModel/models/ForwardingConstruct');
+const ForwardingDomain = require('onf-core-model-ap/applicationPattern/onfModel/models/ForwardingDomain');
 const onfAttributes = require('../../constants/OnfAttributes');
 const fileOperation = require('../../../databaseDriver/JSONDriver');
-const LogicalTerminationPoint = require('onf-core-model-ap-bs/basicServices/utility/LogicalTerminationPoint')
```

Comment:
the same dependency must be also removed from LogicalTerminationPointWithMappingServices.js 
https://github.com/openBackhaul/ApplicationPattern/blob/develop/server/applicationPattern/applicationPattern/onfModel/services/LogicalTerminationPointWithMappingServices.js